src/components/VmsList/style.css

```diff
@@ -1,15 +1,7 @@
-.movable-left { /* general transition properties */
+.vm-card-list-container { /* general transition properties */
```

Comment:
looks like this transition stuff is vestigial, so this entire class can be deleted